docs/pages/eas-update/introduction.mdx

```diff
@@ -15,7 +15,25 @@ All apps running the `expo-updates` library have the ability to receive updates.
 
 Still using Classic Updates? We moved those docs to [the archive](/archive/classic-updates/introduction).
 
-### Get started
+## Pitch
+
+#### Improve User Experience
+
+With EAS Update, you can release hotfixes and enhancements in response to user feedback or market trends, keeping your users engaged and satisfied with your app. Don't wait for the next planned update or manual update request.
+
+#### Broad Compatibility
+
+EAS Update works with vanilla React Native apps, all you need to do is install the lightweight `expo` npm package as a prerequisite, making it a compelling alternative to solutions like [Codepush](/eas-update/codepush).
```

Comment:
```suggestion
EAS Update works with vanilla React Native apps. All you need to do is install the lightweight `expo` npm package as a prerequisite, making it a compelling alternative to solutions such as [Codepush](/eas-update/codepush).
```

Instead of adding a pause, we can split it in two sentences.